### DISCUSSION TITLE
How can I add live wallpapers

### DISCUSSION THREAD

**Discussion by VoIDWALkER7** [prasanthrangan/hyprdots#837] [Q&A]
I want to add gifs as wallpaper for just one particular theme, I tried doing it using a roundabout method, by changing the format of gifs to png and then using them as the wallpaper and that worked for a while, but then there were issues like it was glitching and blurry and all that. Is there a way I can add gifs as wallpaper in hyprdots?

**Comment by RookiexCookie** [comment]
Check this out #440
